### PR TITLE
Side nav bar now covers entire screen on mobile devices. Exit button …

### PR DESCRIPTION
**Pull request by rushil-arun**
Changed the width of the side nav bar on mobile devices to fit the entire screen.

### CODE REVIEW

**Comment by rushil-arun** [comment]
For Issues #8

File: src/components/navbar/navbar.css [hunk 1]
```diff
@@ -143,7 +143,7 @@
   }
 
   #half-bg-side {
```
Comment (ndaeila) [inline src/components/navbar/navbar.css:146]: image
image


For a wider view, like on an iPad or something, can you keep the half-panel, and only show the full-panel for Mobile where it makes sense? Maybe use the sm breakpoint?

Or you can have the panel cover the entire screen as you have, but fix everything to the top right of the screen so it always looks like the contents of the nav bar are right-aligned.

**Review by ndaeila — CHANGES_REQUESTED** [review]
Please see review. Thank you!

**Comment by rushil-arun** [comment]
I adjusted to the side for ipad, but I'm not sure I did it properly dynamically.

**Review by sais14 — COMMENTED** [review]
On mobile view, it still looks like the menu is right-aligned rather than centered.  The ipad/regular web sizes looks right, as it does look like its taking up the entire screen but right aligned.
image

**Comment by rushil-arun** [comment]
@sais14 I think the side nav should always be right aligned; did you mean that it looks centered on mobile?

Thanks for the review!

**Comment by ndaeila** [comment]
> @sais14 I think the side nav should always be right aligned; did you mean that it looks centered on mobile?
> 
> Thanks for the review!

This can wait for the meeting - I want to talk about it in person.

**Comment by rushil-arun** [comment]
Sounds good

**Comment by rushil-arun** [comment]
Moved progress onto new branch (Current Side Nav Bar...). Check for PR from that branch.

**Pull request closed without merging**